src/atoms/Layout/layout.module.scss

```diff
@@ -0,0 +1,96 @@
+@import 'src/assets/stylesheets/global';
+
+$gutter-width: 1.5rem;
+
+@mixin g-unit($cols: 1, $total-col-count: 12) {
```

Comment:
@drewdrewthis I think it would be confusing if we had 2 ways of creating a layout. It might be best to have this component be the source of truth for the grid